terraform/iam.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_iam_role" "ssm_role" {
+  name = "ssm-role"
```

Comment:
follow consistent naming for all of the resources.

Reply:
made the changes